src/Commons/NavBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Navbar from 'react-bootstrap/Navbar'
+import Nav from 'react-bootstrap/Nav'
+
+class NavBar extends React.Component {
+render() {
+      return (
+          <div>
+         <Navbar bg="light" expand="lg">
```

Comment:
Why are you adding one extra node in the tree when simply you can enclose it in <Navbar>?

Reply:
ok